services/dynamic-sidecar/Makefile

```diff
@@ -10,7 +10,7 @@ openapi.json: .env ## Creates OAS document openapi.json
 	@set -o allexport; \
 	source .env; \
 	set +o allexport; \
-	simcore-service-dynamic-sidecar openapi > $@
+	simcore-service openapi > $@
```

Comment:
is it really useful to change this? this shortcut is a bit shaky. I had to remove it from the migration library otherwise sometimes depending on the installation timings, it would point to the wrong binary.

Reply:
my bad changed this due to an error. will revert